eos-payg-csv/tests/eos-payg-csv.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2018 Endless Mobile, Inc.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+#
+# Alternatively, the contents of this file may be used under the terms of the
+# GNU Lesser General Public License Version 2.1 or later (the "LGPL"), in
+# which case the provisions of the LGPL are applicable instead of those above.
+# If you wish to allow use of your version of this file only under the terms
+# of the LGPL, and not to allow others to use your version of this file under
+# the terms of the MPL, indicate your decision by deleting the provisions
+# above and replace them with the notice and other provisions required by the
+# LGPL. If you do not delete the provisions above, a recipient may use your
+# version of this file under the terms of either the MPL or the LGPL.
+
+"""Integration tests for the eos-payg-csv utility."""
+
+import csv
+import os
+import shutil
+import subprocess
+import tempfile
+import unittest
+
+import taptestrunner
+
+
+class TestEosPaygCsv(unittest.TestCase):
+    """Integration test for running eos-payg-csv.
+
+    This can be run when installed or uninstalled. When uninstalled, it
+    requires G_TEST_BUILDDIR and G_TEST_SRCDIR to be set.
+
+    The idea with this test harness is to test the eos-payg-csv utility,
+    its handling of command line arguments, its exit statuses, and basic
+    sanity of its generated output.
+    """
+
+    def setUp(self):
+        self.timeout_seconds = 10  # seconds per test
+        self.tmpdir = tempfile.mkdtemp()
+        os.chdir(self.tmpdir)
+        print('tmpdir:', self.tmpdir)
+	# look for the program in the source dir since it's not built
+        if 'G_TEST_SRCDIR' in os.environ:
+            self.__eos_payg_csv = \
+                os.path.join(os.environ['G_TEST_SRCDIR'], '..', 'eos-payg-csv')
+            self.__datadir = os.environ['G_TEST_SRCDIR']
+        else:
+            self.__eos_payg_csv = os.path.join('/', 'usr', 'bin',
+                                               'eos-payg-csv')
+            self.__datadir = os.path.join('/', 'usr', 'share', 'installed-tests',
+                                          'eos-payg-csv')
+        print('eos_payg_csv:', self.__eos_payg_csv)
```

Comment:
I guess these should be hyphens rather than underscores

Reply:
Fixed locally